crates/rolldown/src/hmr/hmr_manager.rs

```diff
@@ -18,7 +18,7 @@ impl HmrManager {
           changed_modules.push(*module_idx);
         }
         _ => {
-          dbg!("No corresponding module found for changed file path: {:?}", changed_file_path);
+          tracing::debug!("No corresponding module found for changed file path: {:?}", changed_file_path);
```

Comment:
dbg!() doesn't support formatting args

Reply:
I am not sure if here is intent to do like this. @hyf0

Reply:
The rest part lgtm, can you revert this?

Reply:
https://github.com/rolldown/rolldown/pull/3718/files/d24844787e34825ab84f683e4648d010cdb12ed8#diff-dca406b959fe927dfee454b84b90e46c0463228652cc2a436ad7192f31664da9L11, seems this part is intentional